packages/cloudscape-react-ts-website/src/cloudscape-react-ts-website-project.ts

```diff
@@ -59,7 +59,7 @@ export class CloudscapeReactTsWebsiteProject extends ReactTypeScriptProject {
       packageManager:
         options.parent && options.parent instanceof NodeProject
           ? options.parent.package.packageManager
-          : options.packageManager,
+          : NodePackageManager.PNPM,
```

Comment:
Should this be `: (options.packageManager ?? NodePackageManager.PNPM)` to respect the user specified one if provided?

Reply:
done!